### PR TITLE
fix: replace all tag by class selectors

### PR DESCRIPTION
**Pull request by meefox**
Related to https://oat-sa.atlassian.net/browse/TR-1462
PR 2 of 3

To Test.
Full testing can be done with an iPad,
you need to hover on any button during a test run, 
and see it is not triggering popover.

### CODE REVIEW

**Comment by btamas** [comment]
I installed the required extension versions, but I have a js error:
Screen Shot 2021-06-09 at 08 32 49

**Review by btamas — APPROVED** [review]
Looks good to me

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by marpesia — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ]  Commits messages are meaningful
- [ ]  Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by lecosson — APPROVED** [review]
not checked on real hardware (ipad)
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x]  Commits messages are meaningful
- [ ]  Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by jsconan — COMMENTED** [review]
It works, but the changes will have impact on other stakeholders.

**Comment by jsconan** [comment]
> I installed the required extension versions, but I have a js error:
> Screen Shot 2021-06-09 at 08 32 49
> 
> I think it is not related, but strange.

Most probably, you have an outdated tao-core installed

**Comment by meefox** [comment]
> I installed the required extension versions, but I have a js error:
> Screen Shot 2021-06-09 at 08 32 49
> 
> I think it is not related, but strange.

Yes, the problem with old tao-core on version 48.11.0.0 it is good.

File: src/plugins/templates/button-group.tpl [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <li data-control="{{control}}" class="small btn-info btn-group action">
 	{{#each buttons}}
-	<a class="li-inner" href="#" title="{{title}}" data-key="{{@key}}">
+	<span class="li-inner" title="{{title}}" data-key="{{@key}}">
```
[the remark this answers is not in the document]
Comment (meefox) [reply src/plugins/templates/button-group.tpl]: reverted to <a> as discussed using a css prevent solution

**Review by jsconan — APPROVED** [review]
Good job!

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**